jobs/backfill-ghost-explore-follows/index.ts

```
@@ -0,0 +1,368 @@
+                try {
+                    await sendFollowActivity(account, ghostExplore, logger);
+                    processed++;
+                } catch (error) {
+                    failed++;
+                    logger.error('Failed to process account {username}', {
+                        username: account.username,
+                        error,
+                    });
+                }
+                resolve();
+            })
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)